tests/coverage/test_gcovr.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+# return the canonicalized absolute pathname
+realpath() {
+  OURPWD=$PWD
+  cd "$(dirname "$1")"
+  LINK=$(readlink "$(basename "$1")")
+  while [ "$LINK" ]; do
+    cd "$(dirname "$LINK")"
+    LINK=$(readlink "$(basename "$1")")
+  done
+  REALPATH="$PWD/$(basename "$1")"
+  cd "$OURPWD"
+  echo "$REALPATH"
+}
+
+DIR_SRC=$(dirname "$(realpath $0)")
+DATE_TIME=$(date +"%y-%m-%d-%H%M%S")
+DIR_REPORT=$(realpath "gcovr/report_gcovr_${DATE_TIME}")
+DIR_CI=$(realpath "../ci/travis")
+DIR_GTEST=$(realpath "$DIR_CI/googletest")
+GCOVR_REPORT="report.html"
+CTEST_REPORT="ctest.log"
+NUMBER_THREADS="-j 4"
+
+mkdir -p $DIR_REPORT
```

Comment:
move down, where that folder is actually used

Reply:
Ok